norm_script_alpha.py

```diff
@@ -0,0 +1,42 @@
+from fuzzywuzzy import process
+
+def get_normalized(el: str, li: list):
+    reading = process.extractBests(el, li, score_cutoff=75)
+    candidates = [el]
+    best_n = el
+    max_n = len(reading)
+
+    while reading != []:
+        (new,points) = reading.pop()
+        if points < 100 and new not in candidates:
+            candidates.append(new)
+            tmp = process.extractBests(new, li, score_cutoff=75)
+            reading = reading + tmp
+            if len(tmp) > max_n:
+                best_n = new
+    return (best_n, candidates)
+
+
+def write_csv(out: dict):
+    f = open("n_alpha.csv", "w")
+    f.write("Old;Normalized\n")
+    for k in out.keys():
+        for n in out[k]:
+            f.write(n+';'+k+'\n')
+
+if __name__ == "__main__":
+    f = open("brands.txt", "r")
+    li = list(f)
+    f.close()
```

Comment:
In Python we use context managers, such as `with`, to define where a file is open. There is no need to manually close a file, as it is implicit in the `with` context. See also line 21
```suggestion
    with open("brands.txt", "r") as f:
        li = list(f)
```